### ISSUE TITLE
[BUG_SP] Course creation on custom field crashes game.

### ISSUE THREAD

**Issue by bendotcodotuk** [Courseplay/Courseplay_FS22#1243]
**READ FIRST**
https://github.com/Courseplay/Courseplay_FS22/wiki

**Steps to reproduce**
Automatic creation of boundries via ploughing outer edge or drawing boundry on the map.

**Actual behavior**
The boundry line is shown on the map, course creator functions as normal but when generating course FS locks up indefinatly untill Widows displays the 'Not Responding' diologue

**Expected behavior**
Courseplay should idetify the new field boundry and succsessfully create course. 

**Screenshots**
![Screenshot (1)](https://user-images.githubusercontent.com/93056199/159140879-8e72fca1-6f1a-4c0b-9104-cbc77fb0d1a5.png)

**Courseplay version**
7.0.1.6/7.0.1.5

**Log**
Log not creating due to game crashing. (Im somewhat new to this, I'll do what I can to deliver relevent information)

**Map**
Calmsden

**Vehicles/ implements involved**
Which vehicle or implement? In game Valtra, Modhub Kvernland I-plough (mod)
Mod yes/no ?

**Relevant script mods**
Mods like Autodrive or VCA and so on.
Autodrive Active, also REA mods. 

EDIT: Ive removed REA mods and the Course generates after a few minutes. The course that was generated was significantly broken however. After troubleshooting I realised the bypass islands was the cause of the issue, turning this off gave a better result. Applogies for creating a redundant issue, ill leave this information up incase it is of use.

**Comment by Tensuko** [comment]
Log is always created even when the game crashes (patient is the key, even the game hangs up, wait!).

**Comment by Falkwa** [comment]
best way to fix this.   make the field boundries.   save and exit game.   its always that first time if you dont dont save and exit

**Comment by Tensuko** [comment]
This is a workaround and not "the best way to fix this"

**Comment by pvaiko** [comment]
Bypass islands is deactivated automatically when generating a course for a custom field. According to the EDIT in the first post the issue is resolved.

**Comment by bendotcodotuk** [comment]
Appolgies for the delayed reply, had a busy few days.

Before disabling the REA mods the custom field course generation would always end with me having to close FS22 due to a not responding message. After disabling the REA mods the course generated after a few minutes and testing since has been almost instant. This could well be a cpu bottleneck on my end or some for of conflict (which I doubt).

Falkwa: I attempted this a few times over a few days before the reporting the issue and had no effect. 

pvaiko: Yes, I had activated it when generating another course and forgot to turn it off. For what its worth turning the option off solved an issue with an irregular course but did not affect the initial problem of the course not generating at all. The issue is solved as far as I can tell, just wanted to offer as much as I can incase it helps.

If needed here's my log
[log.txt](https://github.com/Courseplay/Courseplay_FS22/files/8346448/log.txt)
.
